pkg/flink/app/application.go

```diff
@@ -66,7 +67,8 @@ func StartApp(gatewayClient ccloudv2.GatewayClientInterface, tokenRefreshFunc fu
 	resultFetcher := results.NewResultFetcher(dataStore)
 
 	// Instantiate LSP
-	lspClient := lsp.NewWebsocketClient(getAuthToken, appOptions.GetLSPBaseUrl(), appOptions.GetOrganizationId(), appOptions.GetEnvironmentId())
+	handlerCh := make(chan *jsonrpc2.Request) // This is the channel used for the messages received by the language to be passed through to the input controller
```

Comment:
one more thing, I think we need to close the channel somewhere. On shutdown maybe?

Reply:
Hmmm, we just create one channel and the only situation where it isn't necessary is when the shell exists, so I don't think this is necessary, right?

That said, I made the change to close it on shutdown it, since it's simple, just to play safe

Reply:
have you? 😄 maybe you haven't pushed, I can't see it

Reply:
good eye, it was a test 👀 pushed it